### PR TITLE
Test Shim: Refactoring various Disappear checks

### PR DESCRIPTION
**Pull request by mbfrahry**

### CODE REVIEW

**Review by katbyte — APPROVED** [review]
LGTM 👍

File: azurerm/internal/services/compute/proximity_placement_group_resource_test.go [hunk 1]
```diff
@@ -109,32 +107,20 @@ func (t ProximityPlacementGroupResource) Exists(ctx context.Context, clients *cl
...
-			return fmt.Errorf("Not found: %s", resourceName)
-		}
-
-		ppgName := rs.Primary.Attributes["name"]
-		resourceGroup, hasResourceGroup := rs.Primary.Attributes["resource_group_name"]
-		if !hasResourceGroup {
-			return fmt.Errorf("Bad: no resource group found in state for proximity placement group: %s", ppgName)
-		}
+func (ProximityPlacementGroupResource) Destroy(ctx context.Context, client *clients.Client, state *terraform.InstanceState) (*bool, error) {
+	id, err := parse.AvailabilitySetID(state.ID)
```
Comment (tombuildsstuff) [inline azurerm/internal/services/compute/proximity_placement_group_resource_test.go:111]: this should be a Proximity Placement Group ID

**Review by tombuildsstuff — CHANGES_REQUESTED** [review]
one thing to fix but this should otherwise be GTG 👍

**Review by tombuildsstuff — APPROVED** [review]
pushed a commit to resolve the proximity placement id parser comment

**Pull request merged**